src/Core/src/Platform/Android/ImageButtonExtensions.cs

```diff
@@ -76,7 +77,9 @@ internal static void UpdateButtonStroke(this ShapeableImageView platformView, IB
 
 		internal static void UpdateButtonBackground(this ShapeableImageView platformView, IImageButton button)
 		{
-			platformView.UpdateMauiRippleDrawableBackground(button,
+			platformView.UpdateMauiRippleDrawableBackground(
+				button.Background ?? new SolidPaint(Colors.Transparent), // transparent to force some background
```

Comment:
Should we use the same default color applied to Button?

Reply:
Not sure this ever was the case, so it may be a breaking change?

Reply:
Let me test OG net8 and see. I am sure I have always seen transparent because this is not a button, it is an image view. I will also test net7.

Reply:
I tested OG 8.0.3 and there is no background. Same with 7.0.101+4.